frontend/packages/insights-plugin/locales/en/insights-plugin.json

```diff
@@ -15,5 +16,5 @@
   "{{issuesNumber}} issue found": "{{issuesNumber}} issue found",
   "{{issuesNumber}} issues found": "{{issuesNumber}} issues found",
   "Insights": "Insights",
-  "Insights status": "Insights status"
-}
\ No newline at end of file
+  "Insights Advisor status": "Insights Advisor status"
```

Comment:
You shouldn't modify this file directly. Let the i18next parser do it using the right comment format

Reply:
Thanks, fixed it with the last push :+1: